README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
```

Comment:
No, right handed is correct. I was just wondering why it was placed in brackets - the way it's written, it looks like active=>right handed.  Those are conceptually distinct though I think?